privacy_policy/en/privacy-notice-website.md

```diff
@@ -0,0 +1,71 @@
+
+Important: If you wish, you can always [disable cookies](http://www.wikihow.com/Disable-Cookies) on your web browsing software. But please note that certain features of our products and services may not function properly without the aid of cookies.
+(/LEARN MORE)
+
+## Interacting with Adguard websites
+### When you visit our website 
+We may use cookies, third-party web analytics, GIFs, device information, and IP addresses for functionality and to better understand the use of our site by our visitors. [LEARN MORE] 
+
+**Analytics & Metrics**: We currently use several website analytics tools. These tools help us understand the use of our site by our visitors by collecting information. 
+
+Piwik (piwik.adguard.com) which we consider the best option from the privacy point of view as data is not shared with third parties (service providers) and we manage it ourselves. In the nearest future we will switch over to this service only. 
+
+In the meantime we as well use Google Analytics and Yandex.Metrika. Users who wish to avoid tracking by Google Analytics and Yandex.Metrika can prevent data collection. 
+
+**To hide yourself:** 
+Install [Google Analytics opt-out extension](https://tools.google.com/dlpage/gaoptout) and [Yandex opt-out add-on](https://yandex.com/support/metrica/general/opt-out.xml).
+We do not track users across third-party websites to provide targeted advertising. If you have enabled “Do Not Track” option when accessing Adguard websites, no tools from Analytics & Metrics will be used.
+(/LEARN MORE)
+
+
+### When you download Adguard products
+You are not required to disclose any personal information in order to download any of our products.
+ 
+### When you purchase Adguard products
+When you pay for Adguard products the billing is generally handled by a third-party service provider. In most cases the service provider is acting as our reseller; thus, you will be making your purchase from the service provider directly, and not from Adguard. The handling of your personal information will be governed by any privacy policy or terms of service published by the service provider. (LEARN MORE)
+
+We save the information you enter on the order page. This information can be used in the future if you request your license key to be recovered.
+
+The third-party service provider may transmit some of your personal information to us, but not including payment information, unless you specifically agree (as may be the case in the event of a purchase with auto-renewal). We may use the information we receive (not including payment information) to verify your registration or license status, to contact you about the status of your account, or for renewal of your subscription, if applicable.  (/LEARN MORE)
+
+### When you leave a comment
```

Comment:
Not just leave. Disqus is used for rendering existing comments. Basically, when you visit a page with disqus comments, they might collect data about you.